NEWS.md

```diff
@@ -152,6 +156,7 @@ Standard library changes
 * The `Pkg.Artifacts` module has been imported as a separate standard library.  It is still available as
   `Pkg.Artifacts`, however starting from Julia v1.6+, packages may import simply `Artifacts` without importing
   all of `Pkg` alongside. ([#37320])
+* `@time` now reports if the time presented included any compilation time, as a percentage ([#37678])
```

Comment:
Hmmm.. I'm not sure of it.

```suggestion
* `@time` now reports the compilation time, if included, as a percentage ([#37678])
```

Reply:
Maybe 
```suggestion
* `@time` now reports compilation time (if any) as a percentage ([#37678])
```

Reply:
Thanks but the original wording is more accurate. It doesn't report total compile time, just any compile time that has contributed to the time reported. For instance, inference time isn't included

Reply:
I think I made it a bit more readable 🤞